privacy-config/privacy-config-api/src/main/java/com/duckduckgo/privacy/config/api/UnprotectedTemporary.kt

```diff
@@ -25,4 +25,9 @@ interface UnprotectedTemporary {
      * otherwise.
      */
     fun isAnException(url: String): Boolean
+
+    /**
+     * This method returns a list of Strings containing the domains in the unprotected temporary exceptions list.
+     */
+    fun allExceptions(): List<String>
```

Comment:
Perhaps rename this to `allExceptionDomains`

Reply:
I've removed this API to use the one you created for content scripts instead :)